tests/test_partial_update.py

```diff
@@ -0,0 +1,42 @@
+import utilities.methods
+from data.testing_data import *
+import pytest
+
+session = utilities.methods.create_session()
+
+
+def test_partial_update_booking():
```

Comment:
Tutaj też bym chciała to sparametryzować, ale nie wiem jak przygotować wtedy uniwersalną asercję, ponieważ np.
1) zmiana imienia + nazwiska
2) zmiana ceny, daty zameldowania i wymeldowania
i inna cześć jest zmieniona, inna pozostaje niezmieniona